lib/paperclip.rb

```diff
@@ -55,6 +55,7 @@
 require 'paperclip/attachment_registry'
 require 'paperclip/filename_cleaner'
 require 'mime/types'
+require 'mimemagic'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.